lib/tools/system-calls.js

```diff
@@ -411,6 +412,20 @@ systemCallMethods.launchAVD = async function (avdName, avdArgs, language, countr
   return proc;
 };
 
+systemCallMethods.getAdbVersion = async function () {
+  if (!_.isUndefined(this.adbVersion)) {
+    return this.adbVersion;
+  }
+  try {
+    let version = await this.adbExec('version');
+    this.adbVersion = version.replace(/(Android\sDebug\sBridge\sversion\s)([\d\.]*)([\s\w\-])*/, "$2");
```

Comment:
Why capture so many groups? Would not `version.replace(/Android\sDebug\sBridge\sversion\s([\d\.]*)[\s\w\-]*/, "$1");` give the same result?